pkg/quarterdeck/db/models/users.go

```diff
@@ -875,7 +875,7 @@ func (u *User) fetchPermissions(tx *sql.Tx) (err error) {
 	return rows.Err()
 }
 
-func (u *User) ToAPI(ctx context.Context) *api.User {
+func (u *User) ToAPI() *api.User {
```

Comment:
removed unnecessary variable - made updates as needed in other places

Reply:
Thanks!